apps/07jquery-tic-tac-toe/script.js

```diff
@@ -2,4 +2,79 @@
 
 $(document).on('ready', function() {
   // Put app logic in here
+  setup();
 });
+
+
+function setup(){
+  mystuff();
+}
+
+function mystuff(){
+
+  // Spec 1
+  // Set a variable playerTurn equal to 'X'.
+  // Using jQuery, set a click listener on all of the [data-cell]s that SETs playerTurn as .text() on $(this) by $(this).text(playerTurn)
+
+  var playerTurn = 'X';
+  $('[data-cell]').click(function(){
+    $(this).text(playerTurn);
+      // var data = $(this).text();
+      // console.log(data + ' here is data');  <---test what is in "this"
+
+    //  console.log(playerTurn)
+    var win =  checkForWin(playerTurn);
+    resetBoard(win);
+
+  //   Spec 2
+  // Also in your click event, toggle playerTurn between 'O' and 'X' using a ternary operator. You should now be able to change turns as you click around on your board.
+    playerTurn = (playerTurn==='X') ? 'O': 'X';  //set playerTurn to X or O
+  }); //end of JQuery function -----------------------
+}// end of mystuff function
+
+//    Reset the board for next game or play
+function resetBoard(){
+  $('#clear').click(function(){
+    var blank = '';
+    var data2 = $('[data-cell]');
+
+    for (var i=0; i<9; i++){
+      data2[i].innerText = '';
```

Comment:
You can also the jQuery method .empty() on the $('data-cell]') object and it will delete the inner text in all of them.

``` Ex: 
$('#clear').click(function(){
   $('[data-cell]').empty(); 
}
```